backend/zfs_operations/services/dataset_service.py

```
@@ -572,4 +573,261 @@ async def _parse_usage_info(self, output: str) -> Result[Dict[str, Any], Dataset
+            match = re.match(r'^(\d+(?:\.\d+)?)\s*([KMGT]?)$', bandwidth_str.upper())
+            if not match:
+                return int(float(bandwidth_str))
+            
+            value = float(match.group(1))
+            suffix = match.group(2)
+            
+            multipliers = {'K': 1024, 'M': 1024**2, 'G': 1024**3, 'T': 1024**4}
+            multiplier = multipliers.get(suffix, 1)
+            
+            return int(value * multiplier)
+        except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)